translator/trace/zipkinv2/to_translator.go

```diff
@@ -378,13 +393,13 @@ func populateResourceFromZipkinSpan(tags map[string]string, localServiceName str
 		return
 	}
 
-	snSource := tags[tagServiceNameSource]
+	snSource := tags[zipkin.TagServiceNameSource]
 	if snSource == "" {
 		resource.Attributes().InsertString(conventions.AttributeServiceName, localServiceName)
 	} else {
 		resource.Attributes().InsertString(snSource, localServiceName)
 	}
-	delete(tags, tagServiceNameSource)
+	delete(tags, zipkin.TagServiceNameSource)
```

Comment:
Please file a bug to ensure we unexport `TagServiceNameSource`. I don't think it needs to be public.